test/NuGetPackagesTests/InstrumentationTargetTests.cs

```diff
@@ -119,21 +119,20 @@ public void InstrumentApplication()
 
     private void ChangeDefaultProgramToHelloWorld()
     {
-        const string ProgramContent = """
-            public static class Program
-            {
-                public static void Main()
-                {
-                    System.Console.WriteLine("Hello World!");
-                }
-            }
-            
-            """;
+        const string ProgramContent = @"
```

Comment:
I tried to suppress dotnet-format around this via the same `#pragma warning disable format` used else where in the code but couldn't: opted to use old style in the end.